ddtrace/appsec/processor.py

```diff
@@ -100,6 +100,17 @@ def _set_headers(span, headers):
             span.set_tag(_normalize_tag_name("request", k), headers[k])
 
 
+def _set_cookies(span, cookies):
+    # type: (Span, Dict[str, Union[str, List[str]]]) -> None
+    for k in cookies:
+        cookie = cookies[k]
+        # flask~=0.12.0 create a list of values for each cookie instead a string
+        if (isinstance(cookie, list) or isinstance(cookie, tuple)) and len(cookie) == 1:
+            cookie = cookie[0]
+        # since the header value can be a list, use `set_tag()` to ensure it is converted to a string
```

Comment:
Just make sure this is the format of the value you actually want, since it'll just be `["cookie", "cookie", "cookie"]`.

if you ever need to process this on the other side, that might not be the best format? (guessing, no idea your actual needs)